projects/incident-language-model.md

```diff
@@ -0,0 +1,24 @@
+---
+remote_theme: nhsx/nhsx-io-theme
+title: Exploring large-scale language models with NHS incident data
+description: NHSX PhD Internship - Incident Language Model
+permalink: /incident-language-model/
+---
+
+# Exploring large-scale language models with NHS incident data
+
+**Keywords:** Natural Language Processing, Healthcare Text, Dataset, Embeddings, Clustering, Neural Networks
+
+**Need:** NHS England and NHS Improvement collect a national repository of incident reporting data for England and Wales.  These data are used to detect emerging patient safety incidents and drive learning that improves patient safety.  Although the as some categorical data fields, the real 'signals' in these data are within free-text field.  Clinical experienced teams review the most servious events, and emerging themes, but the scale of the dataset means that only a few percent can recieve a full review.  Natural Language Processing (NLP) techniques have the potential to unlock learning from data that are not currently reviewed.  The dataset is currently being used for topic modelling and other analyses, but a large scale language model is not yet in use.  The creation of an appropriate language model will allow the use of clustering and other methods to identify novel targets for clinical review.  This project would seek to build the first stages of a language model by applying different representations of text to find preferred models.
+
+**Current Knowledge/Examples & Possible Techniques/Approaches:** Text extraction, processing, vector representations and weighting methods.  Use of modern open source NLP tools and exposure to neural network frameworks such as TensorFlow or Pytorch.
+
+**Related Previous Internship Projects:** n/a as first year of the scheme.  Possibility of architecture already put in place.
+
+**Enables Future Work:** This may lead to ongoing projects of where this work will be the foundation e.g. tuning the selected language model, training on more data, development of production models, examining clusting or anomoly detection algorithms.
```

Comment:
`projects of where` -> `projects where`
`clusting` -> `clustering`
`anomoly` -> `anomaly`